test-it/ExpectedResults/TestVI_ArrayDelete_2.vtr

```diff
@@ -0,0 +1,6 @@
+TestResult_6 true
```

Comment:
Please provide more context related to the case being exercised. Numbers 1 to 6 doesn't ring a bell.

Reply:
Done